src/utils/format.ts

```diff
@@ -56,13 +56,19 @@ export function sanitizeNegativeAndMakeMultipleOf(value?: string | null, default
     : defaultValue
 }
 
-export function validatePositive(value: string, constraint = 0): true | string {
-  return Number(value) > constraint || 'Invalid amount'
-}
-export const validInputPattern = new RegExp(/^\d+\.?\d*$/) // allows leading and trailing zeros
 export const leadingAndTrailingZeros = new RegExp(/(^0*(?=\d)|\.0*$)/, 'g') // removes leading zeros and trailing '.' followed by zeros
 export const trailingZerosAfterDot = new RegExp(/(.*\.\d+?)0*$/) // selects valid input without leading zeros after '.'
 
+/**
```

Comment:
Why are we using RegExp constructor above? How did it get here
Please change to `/(^0*(?=\d)|\.0*$)/g, /(.*\.\d+?)0*$/`